project/core/urls.py

```diff
@@ -21,9 +21,7 @@
 from django.views.static import serve
 from django.views.generic.base import RedirectView
 
```

Comment:
```suggestion
from core.router import CiviWikiRouter
```
You should not delete this line.